AirCasting/Map/CreatingSessionMapView.swift

```diff
@@ -0,0 +1,80 @@
+// Created by Lunar on 23/03/2022.
+//
+
+import UIKit
+import SwiftUI
+import GoogleMaps
+import GooglePlaces
+import Resolver
+
+struct CreatingSessionMapView: UIViewRepresentable {
+    
+    typealias UIViewType = GMSMapView
+    @InjectedObject private var tracker: LocationTracker
+    var isMyLocationEnabled = false
+    
+    init(isMyLocationEnabled: Bool = false) {
+        GMSServices.provideAPIKey(GOOGLE_MAP_KEY)
```

Comment:
I know it was done this way in those PRs I told you about, but actually, we should provide this API key once for the whole app. So I think this could be moved to AppBootstrap class, what do you think @pawelgil ?

Reply:
@jankrzempek , could you move this and delete the duplication from GoogleMapView?